src/Utils.ts

```diff
@@ -1,5 +1,6 @@
-import { Buffer } from '@craftzdog/react-native-buffer';
-import { Buffer as SBuffer } from 'safe-buffer';
+import { Buffer as CraftzdogBuffer } from '@craftzdog/react-native-buffer';
+import { Buffer as SafeBuffer } from 'safe-buffer';
+import { Buffer as FerossBuffer } from 'buffer';
```

Comment:
Is it strictly necessary? It adds an extraneous dependency and on top of that the PR does not add it in the dependencies. Wouldn't it be better to update CraftzdogBuffer upstream so that its method to check isBuffer is the same as FerossBuffer?

Reply:
Agreed.  Removed.  I can still use it in tests, because it's a dependency in the example app.